### DISCUSSION TITLE
Lesspipe broken by tabs

### DISCUSSION THREAD

**Discussion by aeiplatform** [wofr06/lesspipe.sh#134] [Q&A] [answered]
Hello guys,
I have exported BASH_ENV to file which has `tabs 2` in it, but this line breaks the less.

`eval "$(SHELL=/bin/sh /usr/bin/lesspipe)"`

After that, less command doesn't show me original text from regular file.

Why is that so?

**Comment by aeiplatform** [comment]
It seems like `tabs` prints some output...
I guess to move `tabs` into only interactive section...

**Comment by wofr06** [comment, marked as answer]
To me it looks like lesspipe was tried to call in an eval statement (for what reason ?). As lesspipe and not lesspipe.sh was called, it may be unrelated to this version of lesspipe.
